core/src/main/scala/org/apache/spark/MapOutputTracker.scala

```diff
@@ -742,10 +742,10 @@ private[spark] class MapOutputTrackerMaster(
    *                  result.
    */
   def unregisterMergeResult(
-    shuffleId: Int,
-    reduceId: Int,
-    bmAddress: BlockManagerId,
-    mapId: Option[Int] = None) {
+      shuffleId: Int,
+      reduceId: Int,
+      bmAddress: BlockManagerId,
+      mapId: Option[Int] = None): Unit = {
```

Comment:
Note to reviewers: Same here. Was added as part of push-based shuffle.  There was a warning  so fixing it and indentation was off as well